HideAndSeek.js

```diff
@@ -0,0 +1,75 @@
+
+/* display previously-saved stored notes on startup */
+//should return a map in the structure
+/*
+{
+"#9(PR#)": {
+  "files": {
+    "README.md" : sha...(commit)
+  }
+}
+*/
+function loadAllData() {
+  
+}
+
+//add the check boxes to each files files-acations class to make the file is completed or not
+function addCompleteAction() {
+
+}
+
+//This needs to save the data to the local storage
+//should write in this structure
+/*
+{
+"#9(PR#)": {
+  "files": {
+    "README.md" : sha...(commit)
+  }
+}
+*/
+function completeFile(fileName, sha) {
+
+}
+
+//remove a file from the local storage
+function unCompleteFile(fileName) {
+  
+}
+
+//hide the files
+function hideCompletedFiles(fileList) {
+
+}
+
+// check the files for if there is an update
+function filterCompletedFiles(fileMap) {
+
+}
+
+/* Clear all completed files for all pull request from the display/storage */
+function clearAll() {
+
+}
+
+/* Clear all completed files for this pull request from the display/storage */
+function clearPr() {
+
+}
+
+//return the number for the pull request
+function getPullRequest() {
+  
+}
+
+initialize();
+
+function initialize() {
```

Comment:
no docs

Reply:
fixed https://github.com/kyleroush/PR-Hide-And-Seek/pull/1/commits/1314c4c6ce9d8e2ccf3bab37897b828547ed4f4e